src/country-data/country-data.interfaces.ts

```diff
@@ -16,41 +17,72 @@ export interface CountryExtendedData extends CountryRawData {
     neighbors: CountryRawData[]
 }
 
-export class CountryShortSummary {
-    @ApiProperty({ example: 'ARG' })
+
+export class CountryBasicData {
+    @ApiProperty({ description: 'ISO-3 code of the country', example: 'ARG', type: 'string' })
     countryCode: string
 
-    @ApiProperty()
+    @ApiProperty({ description: 'Country name in Spanish', example: 'Argentina', type: 'string' })
     countryName: string
 
-    @ApiProperty()
+    @ApiProperty({ description: 'Country population as of recent estimate', example: 45200000, type: 'number' })
     population: number
+}
+
+class CountryBasicDataPlusInternet extends CountryBasicData {
+    @ApiProperty({ description: 'Main internet domain assigned to this country', example: '.ar', type: 'string' })
+    internetDomain: string
+}
 
-    @ApiProperty()
+export class CountryShortSummary extends CountryBasicDataPlusInternet {
+    @ApiProperty({ description: 'ISO code of the main currency of the country', example: 'ARS', type: 'string' })
```

Comment:
It is possible to add a link to some refence that lists all the legal ISO codes?